src/features/island/collectibles/components/ManekiNeko.tsx

```
@@ -98,11 +100,13 @@ export const ManekiNekoImage: React.FC<Props> = ({ id, open }) => {
   };
 
   useEffect(() => {
-    if (open) {
-      // eslint-disable-next-line react-hooks/set-state-in-effect
-      shake();
-    }
-  }, [open]);
+  if (open) {
```

ESLint

[error] Insert `··`

(prettier/prettier)